Tests/App/src/Acme/AppBundle/DataFixtures/Seeds/ORM/User/user.yml

```
@@ -22,3 +22,16 @@ Victoire\Bundle\UserBundle\Entity\User:
         locale: en
         createdAt: <dateTime('now')>
         updatedAt: <dateTime('now')>
+    luke:
+        username: luke@victoire.io
+        email: luke@victoire.io
+        plainPassword: test
+        firstname: Luke
+        lastname: Skywalker
+        enabled: true
+        locale: en
+        createdAt: <dateTime('now')>
+        updatedAt: <dateTime('now')>
+        roles:
+            ROLE_ADMIN: ROLE_ADMIN
```

Only values will be stored in the database, so I think that we can simplify this line:

```suggestion
            - ROLE_ADMIN
```